cmd/definitions/tmpl/service.tmpl

```diff
@@ -5,7 +5,6 @@ import (
     "context"
     "io"
 
-    "github.com/beyondstorage/go-storage/v4/pkg/credential"
```

Comment:
Ooops, it looks like we need to remove `var _ credential.Provider` below too.

Reply:
Okay, Do I have to run `make build` again?

Reply:
> Okay, Do I have to run `make build` again?

Yes, we need to use `make build` to generate code every time we changed the template.

Reply:
I have removed the `var_credential.Provider`. Please take a look at it and let me know if this needs any changes.
Thanks